packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -63,6 +63,7 @@ class Checkbox extends StatefulWidget {
     required this.onChanged,
     this.mouseCursor,
     this.activeColor,
+    this.fillColor,
```

Comment:
We should assert that only fillColor or activeColor, focusColor, hoverColor are specified.

Reply:
Good call, added asserts to checkbox and radio (already had them on Switch).